src/components/molecules/OnlineStats/OnlineStats.tsx

```diff
@@ -145,14 +148,14 @@ const OnlineStats: React.FC = () => {
   }, [fuseVenues, filterVenueText, venuesWithAttendance]);
 
   const filteredUsers = useMemo(() => {
-    if (filterUsersText === "") return partygoers;
+    if (filterUsersText === "") return recentWorldUsers;
```

Comment:
Maybe:
```jsx
if (!filterUsersText) return recentWorldUsers
```

Reply:
Even though it is not related to this PR, I have made a change since it is really small